### ISSUE TITLE
Lowercased 'bearer' in bearer authentication example.

### ISSUE THREAD

**Issue by ivan-maksin-ss** [Mermade/widdershins#113]
According to https://tools.ietf.org/html/rfc6750 
```
The syntax for Bearer credentials is as follows:

     b64token    = 1*( ALPHA / DIGIT /
                       "-" / "." / "_" / "~" / "+" / "/" ) *"="
     credentials = "Bearer" 1*SP b64token
```

see `2.1.  Authorization Request Header Field`

But example syntax is 
```
authHeader.exampleValues.object = 'bearer {access-token}';
```

**Comment by MikeRalphson** [comment]
Thanks - I was probably thinking about header keys being case-insensitive and carried that across to the value. Will fix, or do you fancy creating a Pull Request to fix it?

**Comment by MikeRalphson** [comment]
Will close when a new release is cut.

**Comment by MikeRalphson** [comment]
Included in v3.2.2 - thanks again.
